scripts/pull_request.sh

```diff
@@ -100,6 +101,6 @@ main () {
 
 echo "==========================================================================
 START: Creating SourceCred Cred Update Pull Request!";
-main;
+main "$@"
```

Comment:
what inputs args go here?

Reply:
From the commit message:

> Shell scripts should invoke `main` as `main "$@"` so that the main
> function can actually read those arguments. We don’t use them now, but
> this kind of error can be pretty confusing to debug later in case we do
> want to use them.

<https://github.com/vsoch/cred-action/pull/4/commits/e153beb11d6b981a08efaa95f1eb636fc0372ca7>

Reply:
That's fair! Good to double check that the intention wasn't to actually have any input args (at least for now).